libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Operations.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive
+{
+    public static class Operations
```

Comment:
>Operations [](start = 24, length = 10)

Hm.  These seem rather generic.  What are these for?  Comments on all of these please.